components/StarkNet/modules/getting_started/pages/environment_setup.adoc

```diff
@@ -1,160 +1,208 @@
+* xref:#_install_the_additional_dependencies[Install the additional dependencies]
+* xref:#_install_the_cairo_cli[Install the Cairo CLI]
+* xref:#_install_the_cairo_compiler[Install the Cairo compiler]
 
-In order to install and use `cairo` and `cairo-lang`, install the following: +
 
-* https://www.python.org/downloads/release/python-390/[Python 3.9] +
-* https://www.rust-lang.org/tools/install[Rust] +
+## Install the prerequisites
 
-== cairo-lang CLI Installation
+We recommend using a link:https://docs.python.org/3/library/venv.html[Python virtual environment]
+for your development work.
 
-=== Step 1: Set up your virtual environment
+To create and enter the Python virtual environment, you have two options:
 
-We recommend working inside a Python virtual environment, but you can also install the `cairo`
-package directly. To create and enter the virtual environment, type:
+* xref:#_option_1__activate_your_cairo_virtual_environment_manually[Activate your Cairo virtual environment manually]
+* xref:#_option_2__use_the_python_environment_manager[Use the Python environment manager]
+
+
+### Option 1 - Activate your Cairo virtual environment manually
+
+Run the following two commands to manually create and activate a Cairo virtual environment:
 
 [,Bash]
 ----
-#Activate your cairo virtual environment
 python3.9 -m venv ~/cairo_venv
 source ~/cairo_venv/bin/activate
 ----
 
-Alternatively, you can also use the `pyenv` environment manager to create a virtual environment. To do so, run the following commands:
+After creating and activating the virtual environment, you should see `(cairo_venv)` in the command line prompt, indicating that the virtual environment is active, for example:
+
+[,Bash]
+----
+(cairo_venv)
+----
+
+
+### Option 2 - Use the Python environment manager
+
+Alternatively, you can use the Python environment manager to create a virtual environment. Simply run the following commands:
 
 [,Bash]
 ----
-#Install pyenv
 curl https://pyenv.run | bash
+----
+
+Now add the following to your `.bashrc` or `.zshrc` file:
 
-#Add the following to your .bashrc or .zshrc
+[,Bash]
+----
 export PATH="$HOME/.pyenv/bin:$PATH"
 eval "$(pyenv init -)"
 eval "$(pyenv virtualenv-init -)"
+----
+
+Restart your terminal and run the following commands:
 
-#Restart your terminal and run the following commands
+[,Bash]
+----
 pyenv install 3.9.0
 pyenv virtualenv 3.9.0 cairo_venv
 pyenv activate cairo_venv
 ----
 
-Either way, make sure the `venv` is activated – you should see (`cairo_venv`) in the command line prompt.
-
-Your terminal should look like this:
+you should see `(cairo_venv)` in the command line prompt, indicating that the virtual environment
+is active, for example:
 
 [,Bash]
 ----
-(cairo_venv) satoshi@nakamoto-notebook % 
-
+(cairo_venv)
 ----
 
-=== Step 2: Install the necessary dependencies
+## Install the additional dependencies
+
+Depending on your operating system, you may need to install the following, additional dependencies:
+|===
```

Comment:
I mentioned to Yuki that table contents should wrap by default, not scroll. You should mention it as well.